apps/web/app/admin/members/_components/Navigation.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable @typescript-eslint/no-confusing-void-expression */
+import { useState } from "react";
+import { MOCK_CATEGORIES } from "../mockData";
+import CategoryTab from "./CategoryTab";
+import SortDropdown from "./SortDropdown";
+
+interface NavigationProps {
+  activeTab: string;
+  onTabChange: (category: string) => void;
+}
+
+export default function Navigation({ activeTab, onTabChange }: NavigationProps): JSX.Element {
+  const [selectedSort, setSelectedSort] = useState("최신순");
```

Comment:
sortOption은 해당 컴포넌트 안에서 상수로 관리해도 될거같아
```ts
const SORT_OPTION = [option1, option2 ...] as const;
```
```suggestion
  const [selectedSort, setSelectedSort] = useState(SORT_OPTION[1]);
```